ui/src/ScholarReader.tsx

```diff
@@ -404,12 +409,41 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
           undefined,
           { name: "XYZ" },
           box.left + SCROLL_OFFSET_X,
-          box.top + SCROLL_OFFSET_Y
+          box.top + SCROLL_OFFSET_Y,
+          null,
         ]
       });
     }
   }
 
+  scrollSymbolIntoView() {
+    const { selectedSymbol, pdfViewer, pages, pdfSideBarIsOpen } = this.state;
+    const PDF_SIDE_PANEL_WIDTH = 200;
+    if (pdfViewer && selectedSymbol) {
+      const symBounds = selectedSymbol.bounding_boxes[0];
+      if (pages[symBounds.page + 1].view != null) {
+        const { left, width } = selectors.divDimensionStyles(
+          pages[symBounds.page + 1].view, symBounds
+        );
+        /*
+        * Each component of the calculation: 
+        * left + width = right position on the pdf page of the selected symbol
+        * scrollLeft = how much the pdf has been scrolled left already
+        * PDF_SIDE_PANEL_WIDTH = the extra padding from the pdf's side panel (hard coded in pdf.js code)
+        * ----------------
+        * innerWidth = possible visible area of the viewport for the entire website
+        * 470 = width of the drawer that is now obscuring the view
+        */
+        const relativeSymbolRightPosition = (left + width) - pdfViewer.container.scrollLeft + (pdfSideBarIsOpen ? PDF_SIDE_PANEL_WIDTH : 0);
```

Comment:
@zkirby the `left` position of the symbol is relative to the left-hand side of the page (not relative to the left-hand size of the PDF viewer). Does that mean you need to add in the page `left` coordinate to this computation of the relativeSymbolRightPosition?

Reply:
Ah hahaha, I am realizing now - I need to do more testing - but I think I've found a solution based on your above comment to remove the pdfSideBar event hahaha

Reply:
Sweet, I look forward to seeing it!